src/components/z-popover/styles.css

```diff
@@ -0,0 +1,204 @@
+:host {
+  font-family: var(--dashboard-font);
+  font-weight: var(--font-rg);
+
+  position: relative;
+  display: inline-block;
+}
+
+.popover-content-container {
```

Comment:
In the Abstact is also requested a default padding of 8px

Reply:
Risolto